docs/user-guide/browsers.md

```diff
@@ -11,16 +11,16 @@ Browsers evolve very rapidly, so this table may become inaccurate as new bugs on
 
 | Browser | Support | Notes |
 |---|---|---|
-| Chromium based | ✅ | We support the current stable - 10 versions |
-| Firefox | ✅ | We support current ESR - 1 versions |
-| Safari | ✅ | We support current - 1 versions |
-| Internet Explorer | ❌ | Are you serious? |
+| Chromium-Based (<i class="fa-brands fa-chrome"></i> Chrome, <i class="fa-brands fa-edge"></i> Edge, etc. ) | ✅ | We Support the Current Stable and Previous **10** Versions |
+| <i class="fa-brands fa-firefox-browser"></i> Firefox | ✅ | We Support the Current Extended Support Release and Previous 1 Version(s) |
+| <i class="fa-brands fa-safari"></i> Safari | ✅ | We Support Current and Previous 1 Version(s) |
+| <i class="fa-brands fa-internet-explorer"></i> Internet Explorer | ❌ | Are you serious? [Internet Explorer Support Ended - See this](https://docs.microsoft.com/en-us/lifecycle/announcements/internet-explorer-11-end-of-support) |
```

Comment:
Drop the extra explanation on IE please, it never supported WebRTC, there was a plugin and we dropped support for that a long time ago.